aws/package.json

```diff
@@ -12,7 +12,7 @@
     "repository": "https://github.com/pulumi/pulumi-cloud",
     "dependencies": {
         "@pulumi/pulumi": "^1.0.0 || ^2.0.0",
-        "@pulumi/aws": "^1.0.0 || ^2.0.0",
+        "@pulumi/aws": "^3.22.0",
```

Comment:
What was the reason we previously allowed `^1.0.0 || ^2.0.0`?  And will requiring `^3.22.0` cause some issue here that we were trying to avoid previously?

Reply:
I see this was partly addressed in the comment at https://github.com/pulumi/pulumi-awsx/pull/624#issue-557864373.

Reply:
This library, believe it or not, was generally compatible with all previous versions of `@pulumi/aws` (at least in the minimal I had previously tried). However, this change introduces an enum that only exists after 3.22.0. It was an oversight that this library didn't also allow ^3.0.0, so I don't anticipate issues from that perspective. There was an issue in `awsx` which has already been addressed going from 2->3 where the tags type became narrower (keys must be string), but I don't expect that to be an issue here.